src/tokenizer.c

```diff
@@ -0,0 +1,503 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   tokenizer.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: chuhlig <chuhlig@student.42.fr>            +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/06/28 20:55:50 by chuhlig           #+#    #+#             */
+/*   Updated: 2024/08/04 14:32:56 by chuhlig          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+#include "token.h"
+
+char	*ft_strncpy(char *s1, char *s2, int n)
```

Comment:
Outsource this.
This would fit into libft.

Reply:
done